src/kvstore/NebulaStore.cpp

```diff
@@ -127,7 +129,7 @@ void NebulaStore::init() {
     }
 
     LOG(INFO) << "Init data from partManager for " << storeSvcAddr_;
-    auto partsMap = partMan_->parts(storeSvcAddr_);
+    auto partsMap = options_.partMan_->parts(storeSvcAddr_);
```

Comment:
MayBe we can delete the "partMan_" field in NebulaStore?

Reply:
Yes, I had removed it.